packages/plugin-ext/src/main/browser/view/tree-views-main.tsx

```diff
@@ -277,6 +288,9 @@ export class TreeViewWidget extends TreeWidget {
         return this._contextSelection;
     }
 
+    protected readonly _onInlineActionEmitter = new Emitter<InlineActionEvent>();
+    readonly onInlineActionEmitter = this._onInlineActionEmitter.event;
```

Comment:
please check naming conventions: https://github.com/theia-ide/theia/wiki/Coding-Guidelines#names and align changed code where applicable

Reply:
fixed